src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:8888/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:8888/project/issues?id=sample&pullRequest=feature1&issues=AWG_n3nvoQnAGKIkmCrP&open=AWG_n3nvoQnAGKIkmCrP)

[ISSUE_KEY]: AWG_n3nvoQnAGKIkmCrP